package-lock.json

```diff
@@ -10208,9 +10208,9 @@
       "dev": true
     },
     "handlebars": {
-      "version": "4.3.0",
-      "resolved": "https://registry.npmjs.org/handlebars/-/handlebars-4.3.0.tgz",
-      "integrity": "sha512-7XlnO8yBXOdi7AzowjZssQr47Ctidqm7GbgARapOaqSN9HQhlClnOkR9HieGauIT3A8MBC6u9wPCXs97PCYpWg==",
+      "version": "4.5.1",
```

Comment:
seems 4.3.0 is also affected.
![Screen Shot 2019-11-05 at 10 23 06](https://user-images.githubusercontent.com/25608386/68195144-9d516a00-ffb6-11e9-8813-cac1d2d65301.png)
 on the advisory, it says to  update to `4.4.5` or later